src/pages/workspaces/workspace/analysis/AnalysisLauncher.js

```diff
@@ -290,7 +312,9 @@ const PreviewHeader = ({
             Nav.goToPath(appLauncherTabName, { namespace, name, application: currentRuntimeTool })
           }
         }, openMenuIcon)],
+      [currentRuntimeTool === tools.Azure.label && runtimeStatus !== 'Running', () => {}],
```

Comment:
It looks like this `cond` returns React elements for other cases. Should this return null instead of an empty object?

Reply:
Ooh. This is returning undefined, not an empty object.